src/product/billOfLading/billOfLadingV1Carrier.ts

```diff
@@ -0,0 +1,72 @@
+
+import { StringDict } from "../../parsing/common";
+import { Polygon } from "../../geometry";
+
+/**
+ * The shipping company responsible for transporting the goods.
+ */
+export class BillOfLadingV1Carrier {
```

Comment:
It might make sense to have these classes extend an abstract class with `toFieldList` as a method that must be implemented.
This was useful in Java when defining methods that would accept these types of classes.
Just an idea for future improvement, not needed for this PR.

Reply:
I like the idea, but that would mean altering some existing classes, and I'd like to avoid it here and now. Noted for the future :+1: